optifit backend/pose_server.py

```
@@ -0,0 +1,315 @@
+            'current_state': analyzer.squat_state
+        }
+    })
+
+
+if __name__ == '__main__':
+    print("="*60)
+    print("Pose Estimation Server Starting")
+    print("Autonomous squat analysis with MediaPipe")
+    print("Rule-driven feedback - no human labeling required")
+    print("="*60)
+    app.run(host='0.0.0.0', port=5001, debug=True)
```

Ruff (0.14.2)

Possible binding to all interfaces

(S104)